### DISCUSSION TITLE
All the achievements?

### DISCUSSION THREAD

**Discussion by erikn69** [Schweinepriester/github-profile-achievements#211] [Show and tell]
https://github.com/Schweinepriester/github-profile-achievements/blob/08c83209c3f082bddf111bb3994a1773913e57a3/README.md?plain=1#L98
@Schweinepriester maybe [Seldaek](https://github.com/Seldaek?tab=achievements)

**Comment by Schweinepriester** [comment]
Dayum, thank you!
Certainly possible they are able max out everything given they have Mars & Arctic.
I‘ll keep an eye on them! :)

**Reply by erikn69** [reply]
@Schweinepriester
[kazupon](https://github.com/kazupon?tab=achievements)
[messense](https://github.com/messense?tab=achievements)
[max-sixt](https://github.com/max-sixty?tab=achievements)

**Reply by Schweinepriester** [reply]
Noice! Each of them so close to 100 % completion :D
